packages/react-core/src/components/Chip/Chip.tsx

```diff
@@ -88,6 +88,7 @@ export class Chip extends React.Component<ChipProps, ChipState> {
     return (
       <Component
         onClick={onClick}
+        {...this.props}
```

Comment:
let's spread the props at the end of the component. That way if the consumer wants to override the `type` or other attributes of the component, they can.